### DISCUSSION TITLE
posts/pure-guide/

### DISCUSSION THREAD

**Discussion by giscus** [rusosnet/rusosnet.github.io#6] [General]
# posts/pure-guide/

Все данные в этом гайде рассчитаны на одного взрослого человека по состоянию на январь 2023. Гайд НЕ относится к несовершеннолетним, лицам с ограниченными возможностями, судимостью на территории РФ (вне зависимости, погашенной или нет).
Цены на услуги и перечень предоставляемых услуг, указанные в гайде, относятся к началу 2023 года и могут быть неактуальными. Чтобы получить информацию о текущих услугах и ценах, наиболее простым способом будет обратиться в общие чаты и сообщить о ваших потребностях.

https://rusos.net/posts/pure-guide/

**Comment by alexesDev** [comment]
Евгения теперь берет 30$/час

**Comment by alexesDev** [comment]
## Дополнение 1

Телефон пополняют на кассах в магазинах.

Симкарту могут дать в отеле. Она активируется по номеру паспорта(загран) и региону (ru)
Но, она не будет работать, пока ее не пополнить.

И еще, не знаю, только у меня так или это везде так.
Первоначально номер можно пополнить только на кассе иагазина или у оператора.
А дальше можно через сайт (antel) (с карты не рф банков)

## Дополнение 2

Мой гайд по симке для приезжих:
 - дойти до пунта каретас шоппинг
- купить prepago claro стимку без паспорта, оператор активирует
- оплачивать криптой на bitrefill.com
